### DISCUSSION TITLE
FNIRSI GC-01: Adding tube protection

### DISCUSSION THREAD

**Discussion by ThatBum** [Gissio/radpro#167] [Q&A]
I've been considering doing this to make experimenting with HV settings safer for the tube and transistors. Would it be as simple as putting a 440V Zener in parallel across the tube?

Given the small amount of current involved, I expect any reasonable size of Zener diode would be able to dissipate the power during breakdown just fine. But is the power supply in this device already current limited? If not, for the sake of the voltage multiplier transistors, would it be recommended to cut a trace to the tube and put a resistor in series with the Zener?

I found [this dubious datasheet](https://github.com/LucasWolfgang/geiger_counter_v3/blob/master/M4011%20Geiger%20Tube%20Specification.pdf) for the M4011 tube claiming its operating current should be between 15 and 20 uA. At 440V that'd call for a 25M ohm resistor.

More to the point, is this even worth it? I have a 1G ohm resistor on the way to measure the voltage properly, so hopefully time spent in an overvoltage/overcurrent state is minimal. Once I find good values I'll probably just set them and not mess with it.

I'm not much of an EE so if anyone has an opinion or advice on implementing this I'd appreciate it. There's a [high resolution picture of the GC-01 board](https://raw.githubusercontent.com/Gissio/radpro/refs/heads/main/docs/devices/FNIRSI%20GC-01/img/FNIRSI%20GC-01.jpg) in the docs.

**Comment by Gissio** [comment]
Let me point you to this discussion: https://www.geigerzaehlerforum.de/index.php?topic=2219.0

**Comment by ugopapetti** [comment]
Hi all,
the schematic in the discussion will control, using an on-off logic, the
production of the pwm input signal to the switching transistor used as
active component for the HV generator by reading the state from an unused
(then dedicated) processor pin.

The good of this approach is the simplicity; additionally the 400V zener
will not allow dangerous voltages to be applied to the Geiger tube.
Differently from a "brute force" zener applied in parallel to the HV to
just limit voltage and protect the tube, the usage of the processor logic
will allow a substantial save in the adsorbed current, i.e. longer working
battery time of the system.
The bad of this approach is that the voltage will be fixed by the zener,
and can not be changed if not replacing the zener with one with one of a
different voltage.

Other approaches, like this one:
geiger_pen/schematic.pdf at main · BleakyTex/geiger_pen · GitHub
<https://github.com/BleakyTex/geiger_pen/blob/main/schematic.pdf>
implementing a triggered voltage comparator will instead allow the
variation of the HV at the cost of a little more complex circuit.

The good of both is that whatever the approach is chosen, the SW logic will
remain the same, i.e one processor input pin to block the HV pwm
generation when the desired HV level is reached (in the voltage comparator
approach the reference voltage can be locally generated with a ref. voltage
component with no other processor pins used).

On both solutions, pwm parameters (duty cycle / frequency) could be tuned
to ensure a stable HV level also for high tube count rates, ensuring
reliable readings also with HI loads, in the meantime when low rates are
counted, the  "strong" pwm signal will be off most of the time.

On both approaches, to reduce the overall current used, R 38 (58MOhm) could
be removed.
Additionally, could be eliminated (to be tested, of course) the dedicated
3.3V regulator (U3 in the GC-01 schematic) using directly 5V or VBat
(however the pulsed current absorbed by the HV circuit could disturb some
operations of the processor, so it is to be carefully tested, as stated
before).

From my point of view, the zener approach is to be preferred, considering
the characteristics of these Chinese counters (fixed tube), and
additionally there will be no need to measure the produced HV value, as
fixed by the zeners.

ugo

Il giorno sab 7 dic 2024 alle ore 19:32 Gissio ***@***.***>
ha scritto:

> Let me point you to this discussion:
> https://www.geigerzaehlerforum.de/index.php?topic=2219.0
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Gissio/radpro/discussions/167#discussioncomment-11495492>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKWI7RALDN3XFYV2T5KZGG32EM5LVAVCNFSM6AAAAABTGH22VSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBZGU2DSMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Ugo Papetti
***@***.***
Tel. 3357368922

**Comment by ThatBum** [comment]
All this is above my pay grade unfortunately, but if anyone wants to attempt this then knock yourself out!
